runtime/battery-station/src/parameters.rs

```diff
@@ -80,6 +80,15 @@ parameter_types! {
     pub const TechnicalCommitteeMaxProposals: u32 = 64;
     pub const TechnicalCommitteeMotionDuration: BlockNumber = 7 * BLOCKS_PER_DAY;
 
+    // Contracts
+    pub const ContractsDeletionQueueDepth: u32 = 128;
+    pub ContractsDeletionWeightLimit: Weight = Perbill::from_percent(10) * RuntimeBlockWeights::get().max_block;
```

Comment:
https://github.com/paritytech/substrate/blob/5ea6d95309aaccfa399c5f72e5a14a4b7c6c4ca1/bin/node/runtime/src/lib.rs#L1151-L1155

Reply:
I decided to deviate here since I want to avoid that extrinsics interacting with the protocol are postponed due to contract deletion taking up the whole block (or blocks). Since contract deletion is low priority, it can be spanned over multiple blocks.